shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -799,10 +828,10 @@ produced ::
   Core.Value era
 produced pp stakePools tx =
   balance (txouts tx)
-    <> ( Val.inject $
-           getField @"txfee" tx
-             <> totalDeposits pp stakePools (toList $ getField @"certs" tx)
-       )
+    <+> ( Val.inject $
+            getField @"txfee" tx
+              <+> totalDeposits pp stakePools (toList $ getField @"certs" tx)
+        )
```

Comment:
any particular reason for this? just don't like the monoid functions? I like the idea of not using custom operators when we don't have to, but I guess it's just all preference. :)

Reply:
There is nothing wrong with the monoid functions, but if we use them it just adds another constriant we must pass around: (Monid(Core.Value era)). Since we already have (Val (Core.Value era)) this keep things simpler.